courses/views_test.py

```diff
@@ -37,7 +40,7 @@ def test_separate_children(self):
         # Different courses
         self.assertNotEqual(m1.course, m2.course)
 
-        resp = self.client.get(reverse('module-list', args=(m1.course.uuid,)))
+        resp = self.client.get(reverse('module-list', args=(m1.course.uuid,)), **self.https)
```

Comment:
Why not make `SSLIFY_DISABLE` an environment variable and then disable it for all tests instead?